src/hooks/useVisibleBalances.ts

```diff
@@ -43,17 +43,17 @@ export const useVisibleBalances = (): {
   loading: boolean
   error?: string
 } => {
-  const balances = useBalances()
+  const data = useBalances()
   const hiddenTokens = useHiddenTokens()
 
   return useMemo(
     () => ({
-      ...balances,
+      ...data,
       balances: {
-        items: filterHiddenTokens(balances.balances.items, hiddenTokens),
-        fiatTotal: getVisibleFiatTotal(balances.balances, hiddenTokens),
+        items: filterHiddenTokens(data.balances.items, hiddenTokens),
+        fiatTotal: data.balances.fiatTotal ? getVisibleFiatTotal(data.balances, hiddenTokens) : '',
```

Comment:
I'm wondering how this condition is fixing it:
The `initialState` of the balances slice contains: `fiatTotal: '',` 
So I think it can never be `undefined`.

Did you find out where the undefined state is coming from?